### DISCUSSION TITLE
Arctic temperature Bias in G-MOM

### DISCUSSION THREAD

**Discussion by whokim** [NCAR/omwg_dev#13] [Biases]
I have generated Arctic temperature bias plots, comparing the two vertical coordinates in G compset (ie., g.e23_a16g.GJRAv4.TL319_t232_zstar_N65.2024.004 and g.e23_a16g.GJRAv4.TL319_t232_hycom1_N75.2024.005) along with a compatible POP 1º simulation. In short, all simulations have a similar degree of cold biases at the Atlantic Water layer (~200-600 m), but Hycom1 appears to have a weaker bias than zstar.

_Eurasian and Canadian Arctic temperature bias relative to WOA18 for MOM and WOA13 for POP as a function of time and depth (3rd and 5th cycles are missing for POP)__

![Screenshot 2024-03-18 at 9 54 41 AM](https://github.com/NCAR/omwg_dev/assets/30511841/008d06ef-2ea8-4341-b968-76712520ac43)

_4th cycle-averaged temperature bias at ~50 m, ~300 m (roughly the core of the Atlantic Water layer), and ~900 m relative to WOA18 for MOM and WOA13 for POP_

![Screenshot 2024-03-19 at 11 44 44 AM](https://github.com/NCAR/omwg_dev/assets/30511841/b380b4a6-1c9b-46be-869d-d9753b3e718a)

**Comment by dabail10** [comment]
Here are some plots from CC Bitz. She is finding that there is a lack of frazil formation just north of Svalbard. Looks like the "heat available" or negative part of frzmlt is very big. On the order of -200 W/m^2. This is preventing ice growth right above the region where the warm salty Atlantic water enters the Arctic Ocean.

![download-5](https://github.com/NCAR/omwg_dev/assets/12142035/cac9953f-403f-41f8-b4d9-4a6142d8882d)
![download-4](https://github.com/NCAR/omwg_dev/assets/12142035/83815dbb-60af-44cd-bcae-0c25af1bf529)
![download-3](https://github.com/NCAR/omwg_dev/assets/12142035/0833391f-d158-4d62-a9fa-208a4fc55fdd)
